package.json

```diff
@@ -1,23 +1,19 @@
 {
   "name": "@thinknimble/tn-forms",
-  "version": "3.2.1",
```

Comment:
after we merge, a new PR will be created by the GH action and will ask to upgrade this to 3.3.0